src/hooks/useTxPendingStatuses.ts

```diff
@@ -76,6 +76,10 @@ const useTxPendingStatuses = (): void => {
   useEffect(() => {
     const unsubFns = Object.entries(pendingStatuses).map(([event, status]) =>
       txSubscribe(event as TxEvent, (detail) => {
+        if (!detail) {
+          return
+        }
+
```

Comment:
Is this related to the fix?

Reply:
Yes, the `detail` of `TxEvent.USER_QUIT` is `undefined`.